### PR TITLE
Speedup

### PR DESCRIPTION
**Pull request by TomasVotruba**
First commit `-34%` :

image

### CODE REVIEW

**Comment by TomasVotruba** [comment]
First commit `-34%` :

image

**Comment by TomasVotruba** [comment]
Second commit -7 %:

image

File: src/FileSystem/FilesFinder.php [hunk 1]
```diff
@@ -49,8 +59,7 @@ private function findInDirectories(array $directories, array $suffixes): array
             ->name($suffixesPattern)
             ->in($directories)
             ->exclude(['examples', 'Examples', 'stubs', 'Stubs', 'fixtures', 'Fixtures', 'polyfill', 'Polyfill'])
```
Comment (carusogabriel) [inline src/FileSystem/FilesFinder.php]: `data`, `Data` can be included. PHPStan uses it: https://github.com/phpstan/phpstan/tree/master/tests/PHPStan/Analyser/data

Comment (TomasVotruba) [reply src/FileSystem/FilesFinder.php]: `data` are safe, but `Data` might be part of the code.

E.g. `DataTranferObjects`

**Comment by TomasVotruba** [comment]
Thanks for review and testing. It matches the results of Blackfire :+1:

**Comment by TomasVotruba** [comment]
Coverage still takes 14 minutes, which is too much over 2 mins of other test runs 
So I drop it for now to keep PR experience smoother: https://travis-ci.org/rectorphp/rector/jobs/416599581#L568

**Comment by TomasVotruba** [comment]
Thanks @JanMikes with help with Blackfire profilling :clap:

**Pull request merged**